internal/microsoft_edge.py

```diff
@@ -68,8 +68,20 @@ def prepare(self, job, task):
         self.bodies_path = os.path.join(task['dir'], 'bodies')
         if not os.path.isdir(self.bodies_path):
             os.makedirs(self.bodies_path)
-        if not task['cached']:
-            self.clear_cache()
+        try:
+            import _winreg
+            keyval = r"SOFTWARE\Classes\Local Settings\Software\Microsoft\Windows\CurrentVersion\AppContainer\Storage\microsoft.microsoftedge_8wekyb3d8bbwe\MicrosoftEdge\Privacy"
+            if not os.path.exists("keyval"):
+                key = _winreg.CreateKey(_winreg.HKEY_CURRENT_USER, keyval)
+            Registrykey = _winreg.OpenKey(_winreg.HKEY_CURRENT_USER, keyval, 0, _winreg.KEY_WRITE)
```

Comment:
The CreateKey should be replaced with CreateKeyEx with KEY_WRITE and OpenKey can be skipped since CreateKeyEx will open the key if it already exists.